CONTRIBUTING.md

```diff
@@ -1,3 +1,70 @@
-Welcome to the Kong/charts repo! We're currently in the process of writing a more detailed CONTRIBUTING guide.
+# Contributing to Kong Helm charts
 
-In lieu of a complete guide, please base your pull requests off the `next` branch and do not increment your chart's version number as part of your changes. The repo maintainers will bundle changes together periodically and bump the version when merging changes from `next` into `master`.
+Feel free to contribute fixes or minor features, we love to receive pull
+requests! If you are planning to develop a larger feature, please come talk to
+us first. We can be found in the [#kong
+channel](https://kubernetes.slack.com/archives/CDCA87FRD) on the Kubernetes
+community Slack instance or the [Kubernetes
+forum](https://discuss.konghq.com/c/kubernetes/19) on Kong Nation.
+
+## Submitting a pull request
+The Kong charts repository accepts contributions via GitHub pull requests to
+the `next` branch.
+
+### Preparing a pull request
+
+Before submitting a pull request, please run through the following steps:
+- Rebase your branch off the current tip of the `next` branch.
+- Run `helm lint` and correct any issues it finds.
+- If your change adds new user-facing (exposed in values.yaml) features or
+  changes existing features, update README.md accordingly. Documentation should
+  adhere to the [Microsoft Writing Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/).
+
+All contributors must sign our Community License Agreement. If you have not yet
+signed it, we will add a comment asking you to do so.
+
+### Review process
+
+Changes to Kong charts undergo automatic and manual review. When you create a
+pull request, CI will run automated tests against a standard set of values.yaml
+configurations. If any fail, you will receive an email alert with details of
+the failure.
+
+If changes are requested or tests find an issue with your changes, please
+add separate fix commits on top of your initial pull request. Do not squash
+changes; the maintainers will squash as needed when merging the pull request.
+
+### Accepted pull requests
```

Comment:
At some point we will have to add a security section to help users contact us with security reports. Not needed in this PR.

Reply:
I believe there plans for a company-wide security reporting system in the not-to-distant future, so yeah, whenever that becomes available.